src/pages/index.tsx

```diff
@@ -11,14 +11,15 @@ import { PageHead } from '../components/PageHead';
 import { CreateBloks } from '../components/CreateBloks';
 import { Layout } from '../components/Layout';
 import { Banner } from '../components/Banner';
+import { resolveRelations } from '../utilities/resolveRelations';
 
 /**
  * This is the layout for the homepage
  */
 
 const IndexPage = ({ data }) => {
   let story = data.storyblokEntry;
-  story = useStoryblokState(story);
+  story = useStoryblokState(story, { resolveRelations });
```

Comment:
After that, it goes into the page templates like this

Reply:
Following this tutorial - different from the old one 😄 
https://www.storyblok.com/tp/create-and-render-blog-articles-in-storyblok-and-gatsby